relay/DCUtR.md

```diff
@@ -0,0 +1,104 @@
+# Direct Connection Upgrade through Relay
+
+| Lifecycle Stage | Maturity      | Status | Latest Revision    |
+|-----------------|---------------|--------|--------------------|
+| 1A              | Working Draft | Active | DRAFT, 2019-05-29  |
+
+Authors: vyzo
+
+Interest Group: raulk, stebalien, whyrusleeping
```

Comment:
Let's follow the [suggested format](https://github.com/libp2p/specs/pull/171/files#diff-3f4c87e0067ecc0fb22bfffc359be3bdR9) here for this section. I know you had concerns about duplicating links, but not everybody reads this spec on github.com.

Reply:
This is still not finalized, and I'll be happy to let yusef be the steward of cosmetics :p

Reply:
Why not do it yourself rather than pull someone else in?

Reply:
I don't like boilerplate, especially in early drafts.
When we are ready to merge we can handle cosmetics.

Reply:
@vyzo: I don't understand why you adopt one part of the "boilerplate" and not another. This whole discussion is borderline childish.

The point of the specs process is to _merge early drafts_, so I was ready to approve this, but can't do so if the author fails to adhere to the spec format deliberately and without fair reason.

Reply:
The header spec hasn't even been finalized yet, there is nothing to adhere to yet.
Regardless I will add your boilerplate since you are so picky about it.